### DISCUSSION TITLE
Release mode returns error occurred when connecting to api while debug mode is fine, Android Platform

### DISCUSSION THREAD

**Discussion by modula2019** [unoplatform/uno#17913] [Q&A]
Hi all,
I don't know if anyone has encountered this before. I can deploy to android device in debug mode and the application worked fine. It is able to connect to the restful API setup to use. When I switch to release mode for production, the application also deploys. However, when it reaches the API calling, it returns errors occurred. I am not able to debug in release mode to be able to know what the cause is. All the permissions needed in Android have been applied. Does anyone have an idea what I should look out for in release mode? 

```
<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
	<uses-permission android:name="android.permission.INTERNET" />
```

I am using uno SDK 5.3.90.
Thanks

**Comment by francoistanguay** [comment]
What is the error? Do you have a repro project? What are you using to connect to the API? Could it be a Linker issue?
